x/wasm/client/cli/query.go

```diff
@@ -17,6 +17,7 @@ import (
 	flag "github.com/spf13/pflag"
 
 	"github.com/CosmWasm/wasmd/x/wasm/types"
+	wasmvmapi "github.com/CosmWasm/wasmvm/api"
```

Comment:
Question to the Go pros: is the export via the `/api` folder good that way? Or should I make this available in the root project `github.com/CosmWasm/wasmvm`?

Reply:
seems like all the imports in wasmd are from the root project. 
maybe just set an alias at root level `var LibwasmvmVersion = api.LibwasmvmVersion`

Reply:
IMHO root level would be the first place to look for this kind of functionality

Reply:
Thank you! Will move to the root. The `./api` package will become internal soon (https://github.com/CosmWasm/wasmvm/issues/333).